peerconnection.go

```diff
@@ -1567,27 +1573,15 @@ func (pc *PeerConnection) Close() error {
 	pc.signalingState = SignalingStateClosed
 
 	// https://www.w3.org/TR/webrtc/#dom-rtcpeerconnection-close (step #11)
-	// pc.ICEConnectionState = ICEConnectionStateClosed
-	pc.iceStateChange(ice.ConnectionStateClosed) // FIXME REMOVE
```

Comment:
We don't need to set this ourselves `pion/ice` sets this to Closed for us

Reply:
I also added a test that confirms that `Closed` is properly set. Users should see no behavior difference